cmd/testnet-reset/main.go

```diff
@@ -94,6 +98,22 @@ func main() {
 		"generator_url":          gen.BaseURL,
 		"generator_access_token": genCore.netTok,
 	}, nil))
+
+	method := "PATCH"
+	url := "https://api.heroku.com/apps/chain-testnet-info/config-vars"
+	r := strings.NewReader(`{"BLOCKCHAIN_ID":"` + resp.BlockchainID + `"}`)
+	client := http.Client{}
+	req, err := http.NewRequest(method, url, r)
+	must(err)
+	req.Header.Add("Accept", "application/vnd.heroku+json; version=3")
+	req.Header.Add("Content-type", "application/json")
+	req.SetBasicAuth(os.Getenv("AUTH_USER"), os.Getenv("AUTH_TOKEN"))
+	resp, err := client.Do(req)
```

Comment:
If we're not setting anything on the http.Client, should we be using http.DefaultClient here instead?